android_key_gen.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
-/Applications/Android\ Studio.app/Contents/jre/Contents/Home/bin/keytool -genkey -v -keystore key.keystore -alias Mykey -keyalg RSA -keysize 2048 -validity 10000
-
+/Applications/Android\ Studio.app/Contents/jre/Contents/Home/bin/keytool \
+    -genkey -v -keystore key.keystore -alias Mykey -keyalg RSA -keysize 2048 -validity 10000 \
+    -dname "CN=TestGuy, OU=MyCompany, O=MyGame, L=MyCity, ST=NC, C=00" \
+    -storepass 123456 -keypass 123456
```

Comment:
replace storepass with these env
$ANDROID_STORE_PASS
$ANDROID_KEY_PASS

consider github secret

Reply:
It is a little bit dirty, because we need to add password into file Config/DefultEngin.ini. I could add a python script to generate a password and read the ini and place it in config file.

Reply:
env is ok, but can not replace the password in Config/DefultEngin.ini without additional script.

Reply:
how about , let user fill Configs ?
don't provide any automatic password

Reply:
because ios would be similar, ios need certifcate + privatekey in keychain access to sign it.
maybe hard to make it automatic

Reply:
Yes, users can fill any key info they want. If users want to generate a key by themselves, they should follow the guide in README by setting up a correct name, organisation, country etc.

But for testing or demo purpose on android, it is better to generate a key automatically. We are very unlikely to change password in Config/DefultEngin.ini.

IOS would be handled in another PR.

Reply:
encode the password is for easy testing, no big harm... Key infomation in Config/DefultEngin.ini is only for easy testing, at least for development, we don't need to config the key info when we build the apk.

Reply:
Of course, we could remove all key information in Config/DefultEngin.ini and delete all key scripts, but ask user just follow the official guide and setup the key. What do you think?

Reply:
ok